packages/tools/assembly/query-engine-tool/index.ts

```diff
@@ -0,0 +1,36 @@
+import {
```

Comment:
@thucpn you can leave out the query engine tool - we won't convert this to WASM (as it has to many dependencies from the core framework)